### PR TITLE
Accuracy

### PR DESCRIPTION
**Pull request by shak360**

### CODE REVIEW

**Comment by shak360** [comment]
I have merge conflicts I needed to resolve using the Pull Request interface.

**Comment by aarmey** [comment]
This branch is all included on accuracy-asm, right? If so can close.

**Comment by shak360** [comment]
Yep, this branch has deprecated. It can be closed and deleted.

**Comment by shak360** [comment]
Didn't realize `accuracy-asm` merged into this branch. This branch `accuracy` just needs to be merged if you think it's ready Dr. Meyer @aarmey .

File: lineage/fitting_distribution.py [hunk 1]
```diff
@@ -33,8 +33,8 @@ def check_dist(data, verbose=False):
     for i, val in enumerate(dist):
         parameters = eval('sp.' + val + '.fit(data, fscale =1)')
 
-        D, p = sp.kstest(data, val, args=parameters)
-
+        _, p = sp.kstest(data, val, args=parameters)
+        del _
```
Comment (aarmey) [inline lineage/fitting_distribution.py]: Why are you deleting `_`? No need to do this anywhere...

**Pull request merged**